pkg/deployer/lib/controller.go

```diff
@@ -180,6 +182,21 @@ func (c *controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, fmt.Errorf("unable to get landscaper context: %w", err)
 	}
 
+	if lsv1alpha1helper.HasOperation(di.ObjectMeta, lsv1alpha1.TestReconcileOperation) {
+
+		if err := c.removeTestReconcileAnnotation(ctx, di); err != nil {
+			return reconcile.Result{}, err
+		}
+
+		logger.Error(nil, "generating a new jobID, because of a test-reconcile annotation")
```

Comment:
Why do we log this on error level? Shouldn't it be info?

Reply:
I change the log level to `Info`.